Add new font <select> in index, move all font files to fonts subdirec…

after2400 opened the pull request:
…tory, update README with source location of additional fonts.

TODO: fix download to work in Firefox.

after2400 committed:
Add new font <select> in index, move all font files to fonts subdirectory, update README with source location of additional fonts.
TODO: fix download to work in Firefox.

smartin015 commented on index.html:
```diff
@@ -1,167 +1,203 @@
-    </body>
+    // Setup user actions
+    $("input,textarea").on('blur', generateSigns);
+    $("input,textarea").on('keypress', (event) => {
```
L178-200 indentation/line separation is a bit weird IMO. Could you please update this to match the style in the rest of the doc?

smartin015 reviewed (changes requested):
Lookin' good - I have just one request on styling towards the end and I'll approve it.

Thanks!

after2400 committed:
Fix mangled JS formatting

after2400 commented:
Fixed (I think). Didn't set PhpStorm to ES6 mode so it didn't like that section of JS.

smartin015 commented on index.html:
```diff
@@ -173,31 +173,27 @@ <h3>Settings:</h3>
+        // Setup user actions
+        $("input,textarea").on('blur', generateSigns);
+        $("input,textarea").on('keypress', (event) => {
+            if (event.which === 13
```
Nit: this should be
```
if (event.which === 13) {
    generateSigns();
}
```

smartin015 reviewed (approved):
Small nit, otherwise LGTM

after2400 committed:
Fix formatting on check for return/enter key. Add document.body.append/removeChild calls to fix download in Firefox

after2400 commented:
Sorry I should have noticed that, but in my defense it was around 2:30am. If it's not too late, pull the latest commit. I fixed that and added a couple of lines to fix the download functionality in Firefox.

smartin015 commented:
Awesome, thanks! And no worries, been there before :)
